build.gradle

```diff
@@ -34,47 +34,30 @@ kotlin {
   sourceSets {
     commonMain {
       dependencies {
-        api 'org.jetbrains.kotlin:kotlin-stdlib-common'
```

Comment:
stdlib is added by default with 1.4, right?

Reply:
Yep